node/lib/openshift-origin-node/model/application_container_ext/environment.rb

```diff
@@ -273,13 +272,18 @@ def validate_ssh_keys(ssh_keys)
 
         # Add user environment variable(s)
         def user_var_add(variables, gears = [])
+          config = ::OpenShift::Config.new
```

Comment:
Generally, if a file uses a library, the file should include the library, in case it is ever loaded without first loading another file that loads the library, or in case that latter file is changed in the future not to require the library.  However, I don't feel strongly enough about it to argue or hold up the PR.